### DISCUSSION TITLE
Markdown field preview option

### DISCUSSION THREAD

**Discussion by Omripresent** [netbox-community/netbox#11249] [Ideas]
With the current textarea option to add markdown formatted text there's no ability to view the resulting output, this forces users to save the same object multiple times until formatting is correct.

I propose to have a toggle similar to context field with the edit/preview options to be able to view the rendered markdown without saving the object.

**Comment by mmfreitas** [comment]
I find myself thinking about this everytime that I use markdown because I have to save the object various times to have the desired output.
You could open a feature request for this, if it is not implemented maybe look for a markdown live preview online...

**Reply by Omripresent** [reply]
I tried using a few online markdown "editors" and could never get consistent results in the way Netbox renders it.
Was going through the proper contribution procedure, hence this discussion.

This should only require some web UI changes, no DB/backend as far as I can tell.
